### DISCUSSION TITLE
Welcome to SimplySql Discussions!

### DISCUSSION THREAD

**Discussion by mithrandyr** [mithrandyr/SimplySql#74] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by saranggm** [comment]
HI how can we import the .sql file like -InputFile to run the schema file,
i want to run schema for postgresql please help

**Reply by mithrandyr** [reply]
Read the file into a variable and pass the variable to `Invoke-SqlUpdate -Query`.  The module focuses on doing one thing very well and lets other powershell cmdlets/modules handle other tasks, like reading text from a file.


```powershell
$pathToFile = "c:\somepath\somefile.sql"
$fileContents = Get-Content -path $pathToFile
Invoke-SqlUpdate -query $fileContents
```

**Reply by saranggm** [reply]
Hi Mithrandyr 

Thanks you so much its working as expected 
really appreciate
